Classes/EventStore/EventStore.php

```diff
@@ -45,8 +45,14 @@ public function get(EventStreamFilterInterface $filter): EventStream
         return $eventStream;
     }
 
-    public function commit(string $streamName, WritableEvents $events, int $expectedVersion = ExpectedVersion::ANY)
+    /**
+     * @param string $streamName
+     * @param WritableEvents $events
+     * @param int $expectedVersion
+     * @return StoredEvent[]
+     */
+    public function commit(string $streamName, WritableEvents $events, int $expectedVersion = ExpectedVersion::ANY): array
     {
-        $this->storage->commit($streamName, $events, $expectedVersion);
+        return $this->storage->commit($streamName, $events, $expectedVersion);
```

Comment:
Event identifier should be optionally known/set in the application to support at-most-once persistence/delivery when the eventstore runs on a different machine.